### PR TITLE
SGX improvements

### PR DESCRIPTION
**Pull request by npmccallum**

### CODE REVIEW

**Comment by haraldh** [comment]
stderr is normally not line buffered for obvious reasons... you also don't have to call flush, e.g.

**Comment by haraldh** [comment]
Although the `debug!` and `debugln!` could call some `flush()` like routine here.

File: internal/shim-sgx/src/handler/mod.rs [hunk 1]
```diff
@@ -106,7 +132,7 @@ impl<'a> Handler<'a> {
                     OP_SYSCALL => h.handle_syscall(),
                     OP_CPUID => h.handle_cpuid(),
                     r => {
-                        debugln!(h, "unsupported opcode: {:?}", r);
+                        debugln!(h, "unsupported opcode: 0x{:04x}", r);
```
Comment (haraldh) [inline internal/shim-sgx/src/handler/mod.rs:135]:
```suggestion
                        debugln!(h, "unsupported opcode: {:#04x}", r);
```

**Review by haraldh — CHANGES_REQUESTED** [review]

**Comment by npmccallum** [comment]
I'm rejecting this PR.

**Pull request closed without merging**